README.md

```diff
@@ -1,73 +1,189 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+<div align="center">
 
-> Description the project.
+  <img src="src/assests/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
+  <h3><b>Space Travelers Hub</b></h3>
 
-## Built With
+</div>
 
-- Major languages
-- Frameworks
-- Technologies used
+<!-- TABLE OF CONTENTS -->
 
-## Live Demo (if available)
+# 📗 Table of Contents
 
-[Live Demo Link](https://livedemo.com)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Space Travelers Hub <a name="about-project"></a>](#covid-19-metrics-webapp-)
+  - [🛠 Built With <a name="built-with"></a>](#-built-with-)
+    - [Tech Stack <a name="tech-stack"></a>](#tech-stack-)
+    - [Key Features <a name="key-features"></a>](#key-features-)
+  - [🚀 Live Demo <a name="live-demo"></a>](#-live-demo-)
+  - [💻 Getting Started <a name="getting-started"></a>](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors <a name="authors"></a>](#-authors-)
+  - [🔭 Future Features <a name="future-features"></a>](#-future-features-)
+  - [🤝 Contributing <a name="contributing"></a>](#-contributing-)
+  - [⭐️ Show your support <a name="support"></a>](#️-show-your-support-)
+  - [🙏 Acknowledgments <a name="acknowledgements"></a>](#-acknowledgments-)
+  - [📝 License <a name="license"></a>](#-license-)
 
+<!-- PROJECT DESCRIPTION -->
 
-## Getting Started
+# 📖 Space Travelers Hub <a name="about-project"></a>
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+>
 
+**This web app was created using React and Redux. It is used to track the Covid-19 metrics update for the Top 20 African countries in view.**
+## 🛠 Built With <a name="built-with"></a>
+- React
+- HTML5
+- Linters
+- GitFlow
+- Redux
+  
+<img src="src/assests/mock.png" alt="logo"   height="auto" />
 
-To get a local copy up and running follow these simple example steps.
+</br>
+
+![](src/assests/view.gif)
+[Back to top](#readme-top)
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://react-redux.js.org/">React Redux</a></li>
+    <li><a href="https://reactrouter.com/">React Router</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Show a list of countries**
+- **Detail page to see the numeber of cases**
+- **input field for searching of countries**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Deployment <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://covid-19-webb-app.netlify.app/)
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
[OPTIONAL] This is confusing. I think you should have just named the video link something like video presentation or any other descriptive name.